File: FlightsApp/Backend/Flights/DTOs/ViewTicketDTO.cs
namespace Flights.DTOs
{
    public class ViewTicketDTO
    {
        public string Id { get; set; }
        public string UserId { get; set; }
        public int Quantity { get; set; }
        public float SummedPrice { get; set; }
        public string DeparturePoint { get; set; }
        public string ArrivalPoint { get; set; }
        public string DepartureTime { get; set; }
        public int Duration { get; set; }

        public ViewTicketDTO()
        {
        }

        public ViewTicketDTO(string id, string userId, int quantity, float summedPrice, string departurePoint, string arrivalPoint, string departureTime, int duration)
        {
            Id = id;
            UserId = userId;
            Quantity = quantity;
            SummedPrice = summedPrice;
            DeparturePoint = departurePoint;
            ArrivalPoint = arrivalPoint;
            DepartureTime = departureTime;
            Duration = duration;
        }
    }
}

File: FlightsApp/Frontend/Flights/src/app/model/viewTicketDto.ts
export class ViewTicketDto{
  Id?:string
  UserId?:number
  Quantity?:number
  SummedPrice?:string
  DeparturePoint?:string
  ArrivalPoint?:string
  DepartureTime?:string
  Duration?:number
  Valid?:boolean
  public constructor(obj?:any){
    if(obj){
      this.Id = obj.id
      this.UserId = obj.userId
      this.Quantity = obj.quantity
      this.SummedPrice = obj.summedPrice
      this.DeparturePoint = obj.departurePoint
      this.ArrivalPoint = obj.arrivalPoint
      this.DepartureTime = obj.departureTime
      this.Duration = obj.duration
      this.Valid = obj.valid;
    }
  }
}